bb-aem-common/src/main/java/com/cognifide/qa/bb/aem/AemCommonModule.java

```diff
@@ -53,4 +58,20 @@ public void init(Properties properties) {
     });
   }
 
+  /**
+   * Provider method that produces a CloseableHttpClient instances configured with URL, login and password
+   * taken from property files.
+   *
+   * @param url      IP address of Author instance, automatically set to author.ip property.
+   * @param login    Author login, automatically set to author.login property.
+   * @param password Author password, automatically set to author.password property.
+   * @return New instance of CloseableHttpClient.
+   */
+  @Provides
+  public CloseableHttpClient getAuthorHttpClient(@Named(AemConfigKeys.AUTHOR_IP) String url,
+      @Named(AemConfigKeys.AUTHOR_LOGIN) String login,
+      @Named(AemConfigKeys.AUTHOR_PASSWORD) String password) {
+    return new HttpClientProvider(login, password, url).get();
```

Comment:
But maybe we could make this configuration less specific?
So that i could use bobcat with HttpClient in some environments that supports username/password? What do you think?
Currently if someone wants to use CloseableHttpClient the import of AEM common is required, right? Do wew want that? I can imagine situation when i could use the HTTP client with some other system under test...
What do you think?